mmengine/hooks/logger_hook.py

```diff
@@ -91,9 +106,16 @@ def __init__(self,
         self.keep_local = keep_local
         self.file_client_args = file_client_args
         self.json_log_path: Optional[str] = None
+
         if self.out_dir is not None:
             self.file_client = FileClient.infer_client(file_client_args,
                                                        self.out_dir)
+            if file_client_args is None:
+                self.file_backend = get_file_backend(
+                    self.out_dir, backend_args=backend_args)
+            else:
+                self.file_backend = self.file_client
+
```

Comment:
```suggestion
        if self.out_dir is not None:
            if file_client_args is None:
                self.file_backend = get_file_backend(
                    self.out_dir, backend_args=backend_args)
            else:
                self.file_backend = FileClient.infer_client(file_client_args,
                                                                                      self.out_dir)

```

Reply:
Keep `self.file_client_args` to avoid bc-breaking. This property is basically not accessed by the caller, so consider removing it to simplify the code.